### PR TITLE
Fix deleting NSSDB objects

### PR DESCRIPTION
**Pull request by Jakuje**
This is mostly my naive approach. Not sure if there is a better way or better place where this could be handled.

### CODE REVIEW

File: src/storage/nssdb/mod.rs [hunk 1]
```diff
@@ -882,6 +886,13 @@ impl Storage for NSSStorage {
                 )?;
             }
         }
+        /* add back the attributes that we requested, but that do not exist in DB */
+        for a in NSS_SKIP_ATTRIBUTES {
+            match attributes.iter().position(|r| r.type_ == a) {
+                Some(_) => obj.set_attr(Attribute::from_bool(a, true))?,
```
[the remark this answers is not in the document]
Comment (Jakuje) [reply src/storage/nssdb/mod.rs:892]: I don't know ... that was something I was considering but did not figure out how to do that yet.

[the remark this answers is not in the document]
Comment (simo5) [reply src/storage/nssdb/mod.rs]: handled

**Comment by simo5** [comment]
Rebased on main and pushed a patch that addresses my request.
Plwase counter-review

**Review by simo5 — APPROVED** [review]

**Review by Jakuje — COMMENTED** [review]
I can not approve my PR, but your changes look good.

**Comment by simo5** [comment]
Great thanks!

**Pull request merged**